.travis.yml

```diff
@@ -2,15 +2,23 @@ language: node_js
 matrix:
   fast_finish: true
 node_js:
-- '12'
+- '16'
```

Comment:
You should use node 12 and 16. 

By default node 12 every where, but you can run test on node 16 also. 

exemple: https://github.com/cozy/cozy-drive/commit/bb6d22441a3d31a23c5817998c990898b977733e